kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -681,6 +682,9 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
         .def("CombineModelParts", [&](ModelPartCombinationUtilities &self, Parameters Param) { return &self.CombineModelParts(Param); }, py::return_value_policy::reference_internal)
     ;
 
+    auto single_model_part_import = m.def_submodule("SingleImportModelPart");
+    single_model_part_import.def("Import", &SingleImportModelPart::Import );
```

Comment:
```suggestion
    single_model_part_import.def_static("Import", &SingleImportModelPart::Import );
```
Not 100% sure but I think it should be `def_static`.

Reply:
OK. It seems that `def_static` works when exporting as classes (not submodules). Sorry for the noise.